app/src/main/res/layout-land/activity_event_initial.xml

```diff
@@ -232,13 +232,6 @@
                             android:layout_height="wrap_content"
                             android:orientation="vertical" />
 
-                        <include
```

Comment:
Why is this deleted in the landscape layout but not in the portrait one?? I think we need it to capture coordinates in the event initial screen.